packages/react-components/src/tag/src/TagList.tsx

```diff
@@ -81,8 +90,9 @@ export function InnerTagList({
     forwardedRef,
     children,
     ...rest
-}) {
-    const nodes = useCollection(children, { items });
+}: InnerTagListProps) {
+    const nodes = useCollection(children);
+    const items = useOnlyCollectionItems(nodes);
```

Comment:
why useCollectionItems? Filtering is useless in this case since it will never be anything else than items. Maybe do an hard cast instead?

Seems weird to call an extra function only for types

Reply:
It used to be items passed as a prop down to useCollection. Its not a valid usecase anymore. I copied the way it was done in other places, useCollection + useOnlyCollectionItems. i might be wrong on how to do it